packages/forms-web-app/src/dynamic-forms/controller.js

```diff
@@ -17,6 +18,15 @@ const {
 } = require('./dynamic-components/views');
 //todo: should this be tied to a particular view, or can this be obtained from Journey object?
```

Comment:
just wanted to flag this up - any thoughts on making the view dynamic as part of the Journey class?

Reply:
I've created that as subtask for Lauren to pick up, so yes but was thinking of getting this in separately